.github/workflows/stop-runner.yml

```diff
@@ -19,17 +27,28 @@ on:
         required: true
       AWS_REGION:
         required: true
+      AWS_ACCOUNT_ID:
+        required: true
 
 jobs:
   stop-runner:
     runs-on: ubuntu-latest
     steps:
-      - name: Configure AWS credentials
-        uses: aws-actions/configure-aws-credentials@v1-node16
+      - name: Checkout 10ds actions repo
```

Comment:
I can see why this copying of other repo's states is so confusing, it has thrown me a couple of times whilst reviewing. Good that we're keeping consistency with the existing approach for now but I'm just glad we've said we'll clean this up 😄